op-node/p2p/peer_params.go

```diff
@@ -0,0 +1,114 @@
+package p2p
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+// TODO: Update these parameters.
+const (
+	slot             = 2 * time.Second
+	epoch            = 12 * time.Second
+	tenEpochs        = 10 * epoch
+	oneHundredEpochs = 100 * epoch
+	decayToZero      = 0.01
+)
```

Comment:
I think the slot/epoch naming is confusing when we don't use it like L1 and when they have different durations and properties.

Can we instead just use L2 blocks as parameter reference?
And I'm afraid we can't make these all constant; if it really depends on the block in terms of duration, then we need to scale it with the rollup config l2 blocktime parameter.